file-formats/scp1/type/zif_aff_scp1_v1.intf.abap

```diff
@@ -0,0 +1,300 @@
+      "! $required
+      field_name  TYPE c LENGTH 30,
+      "! <p class="shorttext">Value</p>
+      "! Value of data record field
+      field_value TYPE string,
+    END OF ty_data.
+
+  "! <p class="shorttext">Data</p>
+  TYPES ty_row_data TYPE SORTED TABLE OF ty_data WITH UNIQUE KEY field_name.
+
+  TYPES:
+    "! <p class="shorttext">Selected Language</p>
+    "! Details of selected language
+    BEGIN OF ty_translation,
+      "! <p class="shorttext">Language</p>
+      "! Language in which the data record is translated
+      language    TYPE c LENGTH 1,
+      "! <p class="shorttext">Field Name</p>
+      "! Name of data record field
+      field_name  TYPE c LENGTH 30,
+      "! <p class="shorttext">Value</p>
+      "! Value of data record field
+      field_value TYPE string,
+    END OF ty_translation.
+
+  "! <p class="shorttext">Translations</p>
+  TYPES ty_translations TYPE SORTED TABLE OF ty_translation WITH UNIQUE KEY language field_name.
+
+  TYPES:
+    "! <p class="shorttext">Data Record</p>
+    "! Details of selected record
+    BEGIN OF ty_record,
+      "! <p class="shorttext">Key Value</p>
+      "! Key value of a record
+      "! $required
+      key_value               TYPE string,
+      "! <p class="shorttext">Record is incomplete</p>
+      "! Automatic Customizing Recording only puts the key fields of a data record or
+      "! data records, in the BC Set. Such BC Sets must be post-processed.
+      "! Such data reocrds are flagged as incomplete
+      "! $values {@link zif_aff_scp1_v1.data:co_incomplete}
+      incomplete              TYPE abap_bool,
+      "! <p class="shorttext">Operation at Activation</p>
+      "! Data records which are to be deleted at activation are flagged with value 'L'
+      "! $values {@link zif_aff_scp1_v1.data:co_operation_at_activation}
+      operation_at_activation TYPE ty_operation_at_activation,
+      "! <p class="shorttext">Data</p>
+      "! Complete data record of selected key value
+      "! $required
+      data                    TYPE ty_row_data,
+      "! <p class="shorttext">Translations</p>
+      "! Translated data records of Table or View included in a BC Set
+      translations            TYPE ty_translations,
+    END OF ty_record.
+
+  "! <p class="shorttext">Data Records</p>
+  TYPES ty_records TYPE SORTED TABLE OF ty_record WITH UNIQUE KEY key_value.
+
+  TYPES:
+    "! <p class="shorttext">Attributes of Table/View Field</p>
+    "! You can define the behavior for field of table/view during activation
+    BEGIN OF ty_field_attribute,
+      "! <p class="shorttext">Field Name</p>
+      "! Name of field in Table/View
+      "! $required
+      field_name      TYPE c LENGTH 30,
+      "! <p class="shorttext">Field Attribute</p>
+      "! Defines the attribute of field like during BC Set activation, the field value must be copied to database
+      "! table or not
+      "! $values {@link zif_aff_scp1_v1.data:co_field_attribute}
```

Comment:
Hint: I think you can move the `$values` annotation to the type definition (in this case to the definition of `ty_additional_info`.

Reply:
I am not sure because to define the constants co_field_attribute, i use type ty_field_attribute_value. So I followed the below sequence to add link on the field level.
1. Type declaration
2. Declare constants using the above type declaration
3. Use type declaration and link to constant in the field

Reply:
Moved the link to type definition.